tests/foreman/ui/test_errata.py

```diff
@@ -726,14 +725,23 @@ def test_host_content_errata_tab_pagination(
 
 @pytest.mark.tier2
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_list(session, function_org_with_parameter, lce, target_sat):
+def test_positive_list(
+    function_org_with_parameter,
```

Comment:
If we look at definition of this function scope fixture `function_org_with_parameter` within same file at line [108](https://github.com/SatelliteQE/robottelo/blob/f2fbe08f5780e42582aad18d817404b960884461/tests/foreman/ui/test_errata.py#L108) it creates SCA disable ord and return, it would not support stream.

Reply:
Performing local testing with a different fixture, SCA-enabled org

Reply:
latest push now uses `module_sca_manifest_org` and `function_org`, thanks for catching!